Removes the shebang line as this script, containing function defintions,
is only imported.
--- a/lib/daps_functions
+++ b/lib/daps_functions
@@ -1,5 +1,3 @@
-#!/bin/bash
-#
 # Copyright (C) 2012-2015 SUSE Linux GmbH
 #
 # Author:
